### PR TITLE
refac: Update C4 model

### PR DESCRIPTION
**Pull request by dstenroejl**
- [x] Add Orchestration function to C4 model
- [x] Show ServiceBus entities (queue/topic/subscription) dependencies

Part of:
https://app.zenhub.com/workspaces/mandalorian-5fd22a1a59e4740018f5ab5a/issues/gh/energinet-datahub/team-mandalorian/201

### CODE REVIEW

File: docs/diagrams/c4-model/model.dsl [hunk 1]
```diff
@@ -1,39 +1,63 @@
 # Read description in the 'views.dsl' file.
 
 wholesaleDomain = group "Wholesale" {
-    wholesaleDataLake = container "Data Lake (Wholesale)" {
-        description "Stores calculation results"
+    wholesaleDataLake = container "Wholesale DataLake" {
+        description "Calculations inputs and results"
```
Comment (JesperGomes) [inline docs/diagrams/c4-model/model.dsl:5]:
```suggestion
        description "Calculation inputs and results"
```

Comment (JesperGomes) [reply docs/diagrams/c4-model/model.dsl:5]: alternatively "Calculations' input and results" or

File: docs/diagrams/c4-model/model.dsl [hunk 2]
```diff
@@ -1,39 +1,63 @@
...
         tags "Microsoft Azure - Azure Databricks" "Mandalorian"
 
-        # Domain relationships
-        this -> wholesaleDataLake "Read / write"
+        # Subsystem relationships
+        this -> wholesaleDataLake "Read inputs / write results"
     }
     wholesaleDb = container "Wholesale Database" {
-        description "Stores calculations and operations data"
+        description "Meta data of calculations and orchestrations"
```
Comment (JesperGomes) [inline docs/diagrams/c4-model/model.dsl:18]: Question: what do you mean by meta data of orchestrations?

Comment (dstenroejl) [reply docs/diagrams/c4-model/model.dsl:18]: Regarding "orchestration meta data" I'm thinking of "status" information that we basically use to understand and control the "state" of the "calculation workflow".

From my point of view it doesn't contain a "calculation", that is why I wrote "meta data".

Bu feel free to just suggest a completely different text. :)

Comment (JesperGomes) [reply docs/diagrams/c4-model/model.dsl:18]: I would just call it "Meta data of calculations". I would say that the "state" is the state of the calculation.

File: docs/diagrams/c4-model/model.dsl [hunk 3]
```diff
@@ -1,39 +1,63 @@
...
```
Comment (JesperGomes) [inline docs/diagrams/c4-model/model.dsl:18]:
```suggestion
        description "Meta data of calculations"
```

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**